schema/data_feed/tracker.fbs

```diff
@@ -49,13 +49,21 @@ table TrackerInfo {
     /// The orientation of the tracker when mounted on the body
     mounting_orientation: solarxr_protocol.datatypes.math.Quat;
 
+    /// Should the tracker's settings be editable by the user
     editable: bool;
 
-    computed: bool;
+    /// Is the tracker computed (solved position and rotation)
+    isComputed: bool;
+
+    /// Is the tracker using an IMU for its tracking data
+    isImu: bool;
```

Comment:
allow_drift_compensation makes this field redundant. If the purpose is to know if drift compensation would be ok, then thats what the `allow_drift_compensation` field is for

Reply:
In fact, we shouldn't be doing drift compensation at all in data feed. Data feed is not the place for any of this.

You can put it in the rpc protocol, as that protocol exposes some implementation details of the server. But the datafeed is not supposed to expose those so this is not the place for it.

If it complicates logic in the server, thats an L, but it can be worked around by using the RPC protocol. Its important to keep this truly generic, the data feed should not reveal anything much about the trackers.

Reply:
> allow_drift_compensation makes this field redundant. If the purpose is to know if drift compensation would be ok, then thats what the `allow_drift_compensation` field is for

No it’s not that, see the note in the description of my PR

Reply:
> In fact, we shouldn't be doing drift compensation at all in data feed. Data feed is not the place for any of this.
> 
> You can put it in the rpc protocol, as that protocol exposes some implementation details of the server. But the datafeed is not supposed to expose those so this is not the place for it.
> 
> If it complicates logic in the server, thats an L, but it can be worked around by using the RPC protocol. Its important to keep this truly generic, the data feed should not reveal anything much about the trackers.

I need the allow_drift_compensation to be unique for each tracker tho.

Reply:
I saw the note. Data feed should not contain this. Instead, please make a rpc request for this information

Reply:
Although, you are right - futura and I had this disagreement before. He said that computed was more of a convenience thing.

Lemme think on this.

Reply:
OK, if you can fix camel case its fine. I'll accept it.

Reply:
Fixed camel case